gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java

```diff
@@ -14,11 +14,13 @@
     private Property<String> url;
     private Property<String> userName;
     private Property<String> userPass;
+    private Property<String> folderPath;
 
-    GatewayConnectionProperties(Project project) {
+    public GatewayConnectionProperties(Project project) {
         url = project.getObjects().property(String.class);
         userName = project.getObjects().property(String.class);
         userPass = project.getObjects().property(String.class);
+        folderPath = project.getObjects().property(String.class);
```

Comment:
Don't revert whole thing.. Story was about moving folderPath to ExportPluginConfig.. You just need public constructor..

Reply:
From what I've initially seen, removing the folderpath from GatewayConnection causes the following issue.

`* What went wrong:
A problem occurred evaluating project ':business-integrations'.
> Could not set unknown property 'folderPath' for object of type com.ca.apim.gateway.cagatewayexport.tasks.export.GatewayConnectionProperties.`

Reply:
@J-Lou There was PR #139 which moved the folderPath from GatewayConnectionProperties to GatewayExportConfig, so defining folderPath there should fix your problem. @amoghsa can you update the wiki for the change? Also there was a soap service change too, can you also update the wiki on that section?

Reply:
Darn, I did know there were docs as well.. I will get to it..

Reply:
Soap service updates have been added..